unity-client/Assets/Scripts/MainScripts/DCL/Controllers/CharacterController/Attachable/AvatarReference.cs

```diff
@@ -0,0 +1,23 @@
+using UnityEngine;
+using DCL.Models;
+using DCL.Controllers;
+using System.Collections.Generic;
+
+public class AvatarReference : Attachable
+{         
+
+    public static AvatarReference i { get; private set; }        
```

Comment:
Not sure if i'm convinced of this singleton setup. We can't put these under an Attachable manager or put the references under another object?

Reply:
Can you point me to another case where we do any of those approaches? Just so I can see how you do it